config/data/langs.toml

```diff
@@ -22,6 +22,24 @@ each argument is NULL terminated.
 x is a no-op.
 '''
 
+[APL]
+args       = [ '/usr/bin/apl', '-' ]
+experiment = 135
+size       = '14.8 MiB'
+version    = 'Dyalog APL 18.2.45405'
+website    = 'https://www.dyalog.com'
+example    = '''
+⍝ Printing
+⎕←'Hello, World!'
+
+⍝ Looping
+⎕IO←0
```

Comment:
This line sets the index origin to zero, just to stay aligned with the samples' output in our other languages.

Reply:
> With respect to `⎕IO` I'd prefer keeping the default. I don't think it's actually that bad. For the holes currently on the site I'd assume the average solution would actually be shorter with `⎕IO=1`, for example Divisors would be ~4 bytes shorter.

Whatever y'all want, I can remove the line and let it behave _defaultish_. Either way works.